spec/requests/tags_spec.rb

```diff
@@ -210,36 +210,37 @@
 
         expect do
           post api_tags_url, :params => { :name => "test_tag", :description => "Test Tag" }
-        end.not_to change(Tag, :count)
+        end.not_to change(Classification, :count)
 
         expect(response).to have_http_status(:forbidden)
       end
 
       it "cannot update a tag" do
         api_basic_authorize
-        tag = Tag.create(:name => "Old name")
+        classification = Classification.is_category.create(:name => "test_tag", :description => "Test Tag")
```

Comment:
How is this different from `FactoryBot.create(:classification, :name => "test_tag", :description => "Test Tag")` ?